app/src/main/java/com/rizadwi/snapsift/presentation/adapter/ArticleAdapter.kt

```diff
@@ -0,0 +1,80 @@
+package com.rizadwi.snapsift.presentation.adapter
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView.Adapter
+import androidx.recyclerview.widget.RecyclerView.ViewHolder
+import com.bumptech.glide.Glide
+import com.rizadwi.snapsift.R
+import com.rizadwi.snapsift.databinding.ItemArticleBinding
+import com.rizadwi.snapsift.model.Article
+import com.rizadwi.snapsift.util.Clipper
+import dagger.hilt.android.qualifiers.ApplicationContext
+import javax.inject.Inject
+
+
+class ArticleAdapter @Inject constructor(
+    @ApplicationContext private val context: Context,
```

Comment:
why passing app context inside the adapter? can we consider to use view context or activity context?

Reply:
hhmm, tbh i've never thought of that. i think it can use view context and it would be better since the context is only used by Glide. i'll do the refactor

Reply:
refactored in PR #18